components/02-molecules/meta/event-meta/yds-event-meta.twig

```diff
@@ -25,56 +25,90 @@
 
 {% set event_meta__base_class = 'event-meta' %}
 
-<div {{ bem(event_meta__base_class) }}>
-  <div {{ bem('inner', [], event_meta__base_class) }}>
-    {# Date #}
-    <div {{ bem('date', [], event_meta__base_class) }}>
-      {# Day #}
-      <div {{ bem('day', [], event_meta__base_class) }}>{{ event_meta__day }}</div>
-      {# Time #}
-      <div {{ bem('time', [], event_meta__base_class) }}>{{ event_meta__time }}</div>
-    </div>
-    {# Event Details #}
-    <div {{ bem('details', [], event_meta__base_class) }}>
-      {# Location #}
-      {% if event_meta__format.0 or event_meta__address %}
-        <div {{ bem('location', [], event_meta__base_class) }}>
-          {# Format #}
-          {% if event_meta__format.0 or event_meta__format %}
-            <p {{ bem('format', [], event_meta__base_class) }}>
-              {% include "@molecules/cards/reference-card/event/_yds-event-format.twig" with {
-                format: event_meta__format,
-              } %}
-            </p>
-          {% endif %}
-          {# Address #}
-          {% if event_meta__address %}
-            <div {{ bem('address', [], event_meta__base_class) }}>
-              {{ event_meta__address }}
+{# Check if event has passed #}
+{% set event_has_passed = event_meta__date_end < "now"|date("Y-m-d") %}
+
+{# Set the event status based on event_has_passed variable #}
+{% set event_status = event_has_passed ? 'passed' : 'current' %}
+
+{# Set the event title prefix based on event status #}
+{% set event_title__prefix = event_has_passed ? 'Past Event: ' : '' %}
+
+<div {{ bem(event_meta__base_class) }} data-event-status="{{ event_status }}">
+  {# To get the event_title__prefix to wire properly to the page_title__prefix
+  and retain the logic of end_date, we need to include the page-title
+  in the same component file #}
+  {# instead of passing the page-title component into a template separately
+  and embedding the meta into it, we need to do that here. #}
+  {% embed "@molecules/page-title/yds-page-title.twig" with {
+    page_title__heading: event_title__heading,
+    page_title__width: 'content',
+    page_title__prefix: event_title__prefix,
+  }%}
+    {% block page_title__meta %}
+      {% if event_has_passed %}
+        <p {{ bem('prefix', [], event_title__prefix__base_class) }}>
+          This event has passed.
+        </p>
```

Comment:
@codeb-a Could you please change  these lines to the following? 

```
        <p {{ bem('past-event', [], event_meta__base_class) }}>
          This event has passed.
        </p>
```
Because this content is part of the event-meta component, it should get the `event_meta__base_class`. 

And then because we changed the selector, we need to update the styles, too. I'll add a comment in that file. 

Apologies, I didn't notice this earlier.